### PR TITLE
Upgrade Python in Dockerfiles to pick up security fixes

### PR DESCRIPTION
**Pull request by harshsharma22**
…021-46848 CVE-2022-1587 CVE-2021-36222

this pr is to fix os vulnerability in pythion lower version .ie 3.8.13

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Comment by harshsharma22** [comment]
@getsentry-release

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.13-slim-bullseye
+FROM python:3.8.16-slim-bullseye
```
Comment (chadwhitacre) [inline docker/Dockerfile:1]: [Looks like](https://peps.python.org/pep-0569/) this is indeed the latest patch release. 👍

**Review by chadwhitacre — APPROVED** [review]
Seems reasonable, thanks @harshsharma22.

Ftr I've confirmed that these are the only two instances in the repo.

```
$ ag 'FROM python'
docker/builder.dockerfile
1:FROM python:3.8.13-slim-buster as sdist

docker/Dockerfile
1:FROM python:3.8.13-slim-bullseye
```

**Comment by asottile-sentry** [comment]
we should also update the development environment at the same time so these aren't out of sync -- there's a few other references throughout the codebase that need updating for that

**Comment by chadwhitacre** [comment]
@asottile-sentry Sounds good, I've disabled automerge.

File: docker/Dockerfile (hunk 1 above)
Comment (evanpurkhiser) [reply docker/Dockerfile:1]: Does this match what we run in sentry.io production?

**Comment by harshsharma22** [comment]
@asottile-sentry  @chadwhitacre  updated the version  in the code base as well as for the development

**Comment by chadwhitacre** [comment]
> we should also update the development environment at the same time so these aren't out of sync -- there's a few other references throughout the codebase that need updating for that

Does a448ebc34219e047bec88337f28b63daa2d6ae9f cover it, @asottile-sentry? Anything else blocking a merge?

**Comment by asottile-sentry** [comment]
> > we should also update the development environment at the same time so these aren't out of sync -- there's a few other references throughout the codebase that need updating for that
> 
> Does [a448ebc](https://github.com/getsentry/sentry/commit/a448ebc34219e047bec88337f28b63daa2d6ae9f) cover it, @asottile-sentry? Anything else blocking a merge?

it should be -- but it needs someone to verify it locally

**Comment by asottile-sentry** [comment]
seems to work locally

**Comment by chadwhitacre** [comment]
Thanks @asottile-sentry! ☺️

**Pull request merged**